### DISCUSSION TITLE
Page organization

### DISCUSSION THREAD

**Discussion by My1** [streamdeck-linux-gui/streamdeck-linux-gui#155] [Ideas / Feature Requests]
Basically 2 things:

1) assign labels to pages for better overview
2) swap pages (which is better when dealing with a "pages in folders"-like setup, as you can reorganize the pages to be together.

**Comment by v4u6h4n** [comment]
I don't know how difficult it is to reorganise pages, but I would second this feature. I have some very complicated layouts, which has resulted in my adding pages I didn't realise I needed that are referenced in early pages and it becomes a bit of a headache when that happens.
